app/src/main/java/nl/tudelft/bbw/controller/BlockController.java

```diff
@@ -48,6 +50,15 @@ public BlockController(User chainOwner, Context context) {
     }
 
     /**
+     *Method to return the own chain
```

Comment:
I have ran it with `gradlew build connectedCheck check lint test` yet I see no problem here. You can try.